packages/data-context/src/DataContext.ts

```diff
@@ -97,6 +97,10 @@ export class DataContext {
     this.lifecycleManager = new ProjectLifecycleManager(this)
   }
 
+  get git () {
+    return this._coreData.currentProjectGitInfo
```

Comment:
```suggestion
    return this.coreData.currentProjectGitInfo
```

I am not sure what, if anything, is different between `_coreData` and `coreData`, but I think we generally want to use the non `_` version.

Reply:
@tgriesser oooh do you know the answer to this question?

Reply:
Yeah I feel like what @lmiller1990 is suggesting sounds right here